docs/src/overview/get-support.md

```diff
@@ -0,0 +1,40 @@
+---
+title: " Get support"
+weight: 4
+toc: false
+description: |
+  Find out how to get help if you're experiencing issues with Platform.sh.
+---
+
+{{< description >}}
+
+
+## Create a support ticket
+
+Create a support ticket when you're experiencing issues related to the proper functioning of the Platform.sh infrastructure, application container software, build processes, possible bugs or if you have general questions.
+
+1. [Open the Management Console](https://console.platform.sh/)
+2. Select **User menu** (your name) > **Support**.
+3. Click **+ New ticket**.
+4. Fill in the ticket fields and click **Submit**.
+
+Or use these shortcuts to [access all support tickets](https://console.platform.sh/-/users/~/tickets) or [open a new ticket](https://console.platform.sh/-/users/~/tickets/open).
+
+Once you submit a ticket, you see it in a list of all tickets created within your organization.
+Note that once you submit the ticket, you won't be able to modify or delete the submission.
+If you have any additional information, you can select the submitted ticket and write a message.
+
+
+
+## Slack
+
+If you want to talk to with engineers and other customers you can join the [public Slack channel](https://chat.platform.sh/).
```

Comment:
```suggestion
To talk with engineers and other customers, join the [public Slack channel](https://chat.platform.sh/).
```

I saw the message from Support both times you posted it. But I think people are coming to this page looking for support, not Support :wink: 

What I think that means is they still want to know why they would want to join here and what kinds of questions might be answered. I might also place the "other customers" first to emphasize that it's not a great way to get immediate contact with Platform engineers.

But mostly, I'd want to emphasize that it's better for questions about app development or whatever that don't fit above but without any expectation that they will necessarily get a reply.